packages/ava/src/utils/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './isType';
 export * from './arr2map';
+export * from './types';
```

Comment:
feel like ts types file

Reply:
But these types do not belong to the util functions here. Should we put it in the root or a `common` directory?

Reply:
to src/common/types.ts